systemtest/src/test/java/io/strimzi/systemtest/AllNamespaceST.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+import io.fabric8.kubernetes.api.model.rbac.KubernetesClusterRoleBinding;
+import io.strimzi.test.annotations.ClusterOperator;
+import io.strimzi.test.annotations.Namespace;
+import io.strimzi.test.extensions.StrimziExtension;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.List;
+
+import static io.strimzi.test.extensions.StrimziExtension.REGRESSION;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.not;
+
+@ExtendWith(StrimziExtension.class)
+@Namespace(AllNamespaceST.CO_NAMESPACE)
+@Namespace(value = AllNamespaceST.SECOND_NAMESPACE, use = false)
+@Namespace(value = AllNamespaceST.THIRD_NAMESPACE, use = false)
+@ClusterOperator
+class AllNamespaceST extends AbstractNamespaceST {
+
+    private static final Logger LOGGER = LogManager.getLogger(AllNamespaceST.class);
+    static final String THIRD_NAMESPACE = "third-namespace-test";
+    private static Resources thirdNamespaceResources;
+
+    /**
+     * Test the case where the TO is configured to watch a different namespace that it is deployed in
+     */
+    @Test
+    @Tag(REGRESSION)
+    void testTopicOperatorWatchingOtherNamespace() {
+        LOGGER.info("Deploying TO in different namespace than CO when CO watches all namespaces");
+
+        kubeClient.namespace(THIRD_NAMESPACE);
+
+        List<String> topics = listTopicsUsingPodCLI(CLUSTER_NAME, 0);
+        assertThat(topics, not(hasItems(TOPIC_NAME)));
+
+        deployNewTopic(SECOND_NAMESPACE, THIRD_NAMESPACE, TOPIC_NAME);
+        deleteNewTopic(SECOND_NAMESPACE, TOPIC_NAME);
+    }
+
+    /**
+     * Test the case when Kafka will be deployed in different namespace than CO
+     */
+    @Test
+    @Tag(REGRESSION)
+    void testKafkaInDifferentNsThanClusterOperator() {
+        LOGGER.info("Deploying Kafka cluster in different namespace than CO when CO watches all namespaces");
+        checkKafkaInDiffNamespaceThanCO();
+    }
+
+    /**
+     * Test the case when MirrorMaker will be deployed in different namespace than CO when CO watches all namespaces
+     */
+    @Test
+    @Tag(REGRESSION)
+    void testDeployMirrorMakerAcrossMultipleNamespace() {
```

Comment:
If we have `KafkaMirrorMaker` test, should we also have `KafkaConnect` test?

Reply:
Okay, it sounds reasonable. I no need to think about virtual machine resources when I create new ST. If we need to check something for a new feature, we need to have ST and no need to care about execution time, resource usage etc

Reply:
Added